src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -12,12 +12,6 @@
     <TargetOS Condition="'$(OSGroup)'!='AnyOS'">$(OSGroup)</TargetOS>
   </PropertyGroup>
 
-  <Target Name="RestoreTestRuntimePackage"
```

Comment:
I would like them to update as well but I also don't want this to be a blocker for people adopting a new build tools version. Perhaps keeping this target and adding another property that allows people to post-pone that work temporarily would be good.